.github/workflows/test.yml

```diff
@@ -616,7 +616,9 @@ jobs:
     - name: Run Tests
       # e2e tests take ~60 minutes normally, 120 should be more than enough
       # set 3 hours for control-plane tests as these might take a while
-      timeout-minutes: ${{ matrix.target == 'control-plane' && 180 || matrix.target == 'control-plane-helm' && 180 || matrix.target == 'external-gateway' && 180 || 120 }}
+      # give 10m extra to give ginkgo chance to timeout before github so that we
```

Comment:
ok so self:notes
CP tests:
each test suite timeout is 180
total test suite timeout is 185
github actions timeout for the CI lane is 190 huh..

shard tests:
each test suite timeout is 120
github timeout is 130